src/components/PurchaseModal/PurchaseModal.tsx

```diff
@@ -91,14 +91,6 @@ const PurchaseModal: FunctionComponent = () => {
           <GridItem span={12}>
             <SectionHeading icon={onlineIcon} title="Online" />
           </GridItem>
-          <GridItem span={6}>
-            <PurchaseCard
-              body="A simpler way to buy and manage enterprise software, with automated deployment to any cloud."
-              ctaHref="https://marketplace.redhat.com"
-              ctaText="Go to the Marketplace"
-              title="Red Hat Marketplace"
-            />
-          </GridItem>
           <GridItem span={6}>
```

Comment:
Should we make this one `span={12}` so that it fills all the space? Maybe worth asking UX about

Reply:
So I played around with span 12 but didn't match the mockup on the comment John left , I can double check though to make sure.

Reply:
Ah, I didn't see a mockup linked on the ticket - do you have a link to it? If that's how its designed then that's perfect!